src/auction.js

```diff
@@ -363,9 +363,12 @@ export function newAuction({adUnits, adUnitCodes, callback, cbTimeout, labels, a
     }
   }
 
-  function addWinningBid(winningBid) {
+  function addWinningBid(winningBid, aUnits = adUnits, am = adapterManager) {
+    const { adUnitCode } = winningBid;
+    const winningAd = aUnits.find(adUnit => adUnit.code === adUnitCode);
```

Comment:
I think `transactionId` is a better ID than `code`. Multiple adUnits can have the same code.